src/api/routes/rosetta/construction.ts

```diff
@@ -165,6 +167,24 @@ export function createRosettaConstructionRouter(db: DataStore, chainId: ChainID)
       }
     }
 
+    // dummy transaction to calculate size
+    const tokenTransferOptions: UnsignedTokenTransferOptions = {
```

Comment:
```suggestion
    // dummy transaction to calculate size
    const dummyTokenTransferTx: UnsignedTokenTransferOptions = {
```

Could rename this so it's clear it's not needed as a real tx